### PR TITLE
Create lerp.mdx

### PR DESCRIPTION
**Pull request by kbinreallife**

### CODE REVIEW

File: docs/scripting/scripts.lib/lerp.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Description
+
+LERP stands for Linear Interpolation; a mathematical function in which a value interpolates between 'start' and 'end' values.
+Formula: start + (end - start) \* t
+
+## Syntax
+
+```js
+return #fs.scripts.lib().lerp(5, 0, 10) // Result will be 10
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/lerp.mdx:13]:
```suggestion
return #fs.scripts.lib().lerp(0.5, 0, 10)
```

File: docs/scripting/scripts.lib/lerp.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+The starting value.
+
+#### End
+
+The ending value.
+
+#### t
+
+The interpolation factor, usually between 0 and 1. If t is 0 the result is equal to Start. When the value is 1, the result is equal to End. Values between 0 and 1 will interpolate between Start and End.
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/lerp.mdx:28]: this is the first parameter. Move it up

File: docs/scripting/scripts.lib/lerp.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### Return
+
+Returns a number.
+
+```
+10
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/lerp.mdx:35]:
```suggestion
5
```

File: docs/scripting/scripts.lib/lerp.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Example
+
+```js
+function ( context, args ) {
+  let l = #fs.scripts.lib();
+  return l.lerp( 5, 0, 10 );
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/lerp.mdx:43]:
```suggestion
function(context, args) {
  const l = #fs.scripts.lib();
  return l.lerp(0.5, 0, 10);
```

**Review by ellipsisasterisk — APPROVED** [review]

**Review by ellipsisasterisk — APPROVED** [review]

**Pull request merged**